ConsoleLog.txt

```diff
@@ -0,0 +1,632 @@
+PS E:\MAX\EPAM\AT_Mentoring_Java_Basic\Java Testing with JUnit\Calculator> mvn clean  test                                
```

Comment:
In the next modules please add gitignore file, that filters report and log files, we shouldn't keep files like surefire_report.7z in our remote repository.

Reply:
Done